tests/validator_churn.go

```diff
@@ -102,6 +104,10 @@ func ValidatorChurnGinkgo() {
 		ctx, subnetBInfo.ChainIDInt, fundedKey, subnetBInfo.ChainWSClient,
 	)
 	Expect(err).Should(BeNil())
+	err = subnetEvmUtils.IssueTxsToActivateProposerVMFork(
```

Comment:
can we iterate through all subnets to issue txs, instead of separate calls for each subnet in case we add subnets down the line.